### PR TITLE
Translated basic pages for pt_BR (Brazilian Portuguese)

### PR DESCRIPTION
**Pull request by lazpeng**
I did not had time to translate the remaining pages but I'll do it soon. I want at least have the basic translated pages to my language, since it'll help a lot on spreading the usage of Rust in Brazil.

### CODE REVIEW

File: _includes/pt-BR/set_platform.js [hunk 1]
```diff
@@ -0,0 +1,49 @@
+(function () {
```
Comment (steveklabnik) [inline _includes/pt-BR/set_platform.js]: ah, it's a shame that these kinds of files currently need to be duplicated. Maybe we can get around that somehow?

File: _layouts/redirect.html [hunk 2]
```diff
@@ -9,6 +9,8 @@
       var languages = {
         "en": "en-US",
         "en-US": "en-US",
+        "pt": "pt-BR", // For now, since there's no pt-PT page, redirect to pt-BR
+        "pt-BR": "pt-BR",
```
Comment (steveklabnik) [inline _layouts/redirect.html]: :smile:

File: _includes/pt-BR/set_platform.js (hunk 1 above)
Comment (lazpeng) [inline _includes/pt-BR/set_platform.js]: I thought the same. I focused on translating so far, but I'll try implementing something like a JIT code generator.

**Comment by lazpeng** [comment]
There's anything more I should add/change? I'll translate the remaining pages, but most of them are big and will demand time, so having these basic stuff available on my language would be great. Thanks!

**Comment by steveklabnik** [comment]
Thanks so much for this!

There's some stuff here that's duplicated that's not great, but that's not your fault; that's stuff that comes up because this is the first one! As such I'm going to _merge_ this for now, and might work on some of that refactoring myself.

So excited! :smile:

**Comment by steveklabnik** [comment]
@lazpeng if I wanted to tweet about this from the Rust account, what's the best way to phrase it in `pt-BR`? :smile:

**Comment by lazpeng** [comment]
@steveklabnik Thanks for sharing it! What exactly would you write? Something like "The Rust website is now available in Brazilian Portuguese, docs coming soon" translates to "O site de Rust está agora em Português, com documentação vindo em breve". Tell me if you'd like to say something else. I'll create some tutorials or start translating rustbyexample soon. Thanks.

**Comment by steveklabnik** [comment]
Done :+1: :heart:

**Pull request merged**